watcher/setup.sh

```diff
@@ -1,19 +1,24 @@
 #!/usr/bin/env bash
 
-# Source this file when starting the container to set it up
+## Source this file when starting the container to set it up
```

Comment:
Why are we keeping this around?

Reply:
Same as before... we still actually have a mix of systems delivered with both mechanisms. I would like to get a couple more miles on the deployment environment before removing these completely.  I guess git history would save us in case things go south but you know, better safe then sorry. 🤔